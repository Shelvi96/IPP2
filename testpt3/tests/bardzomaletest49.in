NEW ID

8991706;>90299:;0
289:>37575
9:6>876055029
0743;>97
8>534002
:14:8:;>73859
4633752>5
70280>2::6
6944>711
192>4:
7075777>;4112021
@555:;:0001523
@62921490131738806690402
@169690:3679;3;3825455
@9;:6;3;5;00161588076;25
@;707
@9;18:;05:98:998:3
@7731:6:599941636755;248326438
@73
@3859:26702069653049
@50:355;41734705:23525
@128:983;80;738465518118
@;:185:6:5382331035;:535:1::6;
@9:1:;615460408820;;3:4:
@73
@5254;7516324;08314;3836;1:2895
@3;:;:680
@;68:751815328159;
