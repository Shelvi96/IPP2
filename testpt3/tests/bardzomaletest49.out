1
430467210
504921625
28532762918
0
4096
54724775604884077
0
1201200
48367701
72708714000
1855562054680170
28314862125
0
2605284679974912000
0
11112
